src/main/java/graphql/execution/instrumentation/Instrumentation.java

```diff
@@ -0,0 +1,77 @@
+package graphql.execution.instrumentation;
```

Comment:
InstrumentationContext<T> allows for callback that are site specific and not casting of Object is needed.  eg  the parse call back gets the parsed document